include/merlin/types.cuh

```diff
@@ -28,6 +28,7 @@ struct Meta {
 };
 
 constexpr uint64_t EMPTY_KEY = 0xFFFFFFFFFFFFFFFF;
+constexpr uint64_t RECLAIM_KEY = 0xFFFFFFFFFFFFFFFE;
```

Comment:
What's the purpose of RECLAIM_KEY. Seems it only changes value when replacement happens in full bucket.

Reply:
It's used in the `core.cuh`